### ISSUE TITLE
`content` attribute of HTTP requests is not documented anywhere

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6181]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6181|
|Type|defect|
|Created|2012-11-12 13:07:26Z|
|Branch|https://github.com/twisted/twisted/tree/request.content-6181|

If you want to access the body of e.g. a POST, you do so with `content` attribute of the `twisted.web.server.Request` instance. This attribute is not, however, documented in the API docs, nor is it mentioned in the "Using Twisted Web" or "Twisted Web in 60 Seconds" howtos.

<details><summary>Searchable metadata</summary>

```
trac-id__6181 6181
type__defect defect
reporter__itamar itamar
priority__normal normal
milestone__ 
branch__branches_request_content_6181 branches/request.content-6181
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1352725646000000 1352725646000000
changetime__1384344304000000 1384344304000000
version__None None
owner__exarkun exarkun
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

[#6195](https://github.com/twisted/twisted/issues/6195) was a duplicate of this.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [36459]) Branching to 'request.content-6181'

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set owner to @exarkun|
|-|-|

 * 
```
+<li><a href="other-request-bodies.xhtml">Other Request Bodies</a></li>
```

I don't think we should capitalize this, "Other request bodies" is fine.

Please merge!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

(In [36483]) Un-titlecase the title link, for local consistency

refs [#6181](https://github.com/twisted/twisted/issues/6181)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [36484]) Merge request.content-6181

Author: exarkun
Reviewer: therve
Fixes: [#6181](https://github.com/twisted/twisted/issues/6181)

Add documentation for the `content` attribute to `twisted.web.iweb.IRequest`.  The
request object has always had this attribute.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

This was a duplicate of [#4252](https://github.com/twisted/twisted/issues/4252).
